components/renku_data_services/project/blueprints.py

```diff
@@ -231,4 +288,5 @@ def _dump_project(project: project_models.Project) -> dict[str, Any]:
             description=project.description,
             etag=project.etag,
             keywords=project.keywords or [],
+            template_id=str(project.template_id) if project.template_id else None,
```

Comment:
Issue: I don't think `str()` is needed here, same a few lines higher. The code in project/apispec_base.py takes care of that. https://github.com/SwissDataScienceCenter/renku-data-services/blob/main/components/renku_data_services/session/apispec_base.py#L20 has a more generic version of that that does it for all fields, but hasn't been ported to other parts of the codebase.